src/main/java/de/dennisguse/opentracks/services/announcement/VoiceAnnouncementUtils.java

```diff
@@ -203,6 +210,17 @@ static Spannable createStatistics(Context context, TrackStatistics trackStatisti
             appendCardinal(builder, context.getString(R.string.sensor_state_heart_rate_value, currentHeartRate), currentHeartRate);
             builder.append(".");
         }
+        if (shouldVoiceAnnounceMaxSlope()) {
```

Comment:
You put it in the wrong method, should be createAfterRecording instead

Reply:
Put it in the right location.